docs/sphinx/methods/dsrg.rst

```diff
@@ -791,6 +792,13 @@ Max macro iterations for MR-DSRG reference relaxation.
 * Type: integer
 * Default: 15
 
+**DSRG_DUMP_RELAXED_ENERGIES**
+
+Save energies that have been pushed to Psi4 environment for all reference relaxation steps.
```

Comment:
I didn't understand this description on first reading. Change to "Dump the energies after each reference relaxation step to JSON?"